### DISCUSSION TITLE
sort legend by item name

### DISCUSSION THREAD

**Discussion by edwardchenchen** [grafana/grafana#63692] [Dashboards, Panels & Visualizations]
I'd like to see the legend in alphabetical order, so that nodes are properly grouped together
![image](https://user-images.githubusercontent.com/6397192/221074467-3c645ae0-b03e-496e-a76e-88f3facf2ecb.png)
I'm getting the data using k8s node exporter
`rate(node_network_receive_bytes_total{job="node-exporter", device="enp0s3"}[$__rate_interval]) * 8`
![image](https://user-images.githubusercontent.com/6397192/221074692-e73f2693-b20e-48c9-a0df-e27e95510079.png)

I have looked into https://github.com/grafana/grafana/pull/40226, but it seems it is sorting by value, not the legend name itself,

https://github.com/grafana/grafana/issues/30907 Time Series: Change legend order
I found this issue is closed, but I did not see how to do this anywhere

related https://github.com/grafana/grafana/issues/18186

I have tried tooltip ordering, but I can only do order by values

So, I'd like to know,
Is it already implemented? if is how can I do it.
if not, is there a feature request?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
